bedrock/externalpages/templates/externalpages/pocket/add.html

```diff
@@ -0,0 +1,116 @@
+         </div>
+         <div class="add-device__context-stores">
+            <h3>Get Pocket for:</h3>
+            <ul class="apps-list">
+               <li class="app-store-badge"><a href="https://apps.apple.com/app/read-it-later-pro/id309601447" target="_blank" rel="noopener noreferrer" class="dark-inline-link">iOS Store</a></li>
+               <li class="app-store-badge"><a href="https://apps.apple.com/app/pocket/id568494494?ls=1&mt=12" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Mac App Store</a></li>
+               <li class="app-store-badge"><a href="https://play.google.com/store/apps/details?id=com.ideashower.readitlater.pro" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Google Play Store</a></li>
+               <li class="app-store-badge"><a href="https://www.amazon.com/dp/B0057PAY8G" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Amazon Store</a></li>
+            </ul>
+            <ul class="device-list">
+               <li><a href="https://chrome.google.com/webstore/detail/save-to-pocket/niloccemoadcdkdjlinkgdfekeahmflj?hl=en" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Windows <span class="rarrow">›</span></a></li>
+               <li><a href="https://help.getpocket.com/article/1006-getting-started-with-kobo" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Kobo e-reader <span class="rarrow">›</span></a></li>
+               <li><a href="https://getpocket.com/apps/windowsphone/" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Windows Mobile <span class="rarrow">›</span></a></li>
+               <li><a href="https://getpocket.com/apps/blackberry/" target="_blank" rel="noopener noreferrer" class="dark-inline-link">Blackberry <span class="rarrow">›</span></a></li>
+               <li><a href="https://getpocket.com/apps/" target="_blank" rel="noopener noreferrer" class="dark-inline-link">More Apps <span class="rarrow">›</span></a></li>
+            </ul>
+         </div>
+      </div>
+   </section>
+
+   <section class="add-save">
+      <div class="add-save__wrapper">
+         <div class="add-save__title">
+            <h2>Unlimited Ways to Save</h2>
+         </div>
+         <div class="add-save__context">
+            <div class="add-save__context-email">
+               <h3>Save via e-mail</h3>
+               <p>email any link to your list by sending it to <a href="mailto:add@getpocket.com" class="dark-inline-link">add@getpocket.com</a> </p>
+               <div class="add-save__context-image">
+                  <img src="{{ static('img/externalpages/pocket/save-via-email.png') }}" alt="">
+               </div>
+               <a
+                  href="https://help.getpocket.com/article/1020-saving-to-pocket-via-email" target="_blank" rel="noopener noreferrer" class="dark-inline-link"
+                  >Learn more</a
+                  >
```

Comment:
There are a few places like this where I think it would be worth tidying up on formatting.